Fitbit.Portable/OAuth2/OAuth2AutoRefreshInterceptor.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Fitbit.Models;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Fitbit.Api.Portable.OAuth2
+{
+    public class OAuth2AutoRefreshInterceptor : IFitbitInterceptor
+    {
+        private const string CUSTOM_HEADER = "X-Fitbit.NET-StaleTokenRetry";
```

Comment:
Rename to remove the X- portion as per Jeremiah's recommendation http://tools.ietf.org/html/rfc6648